app/code/Transoft/Blog/Controller/Adminhtml/Index/Upload.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Please provide ACL resource

Reply:
fixed

Reply:
Am I missed something. I can't see any **blog** related ACL specified

Reply:
removed this action